src/.nuget/dir.props

```diff
@@ -163,6 +179,11 @@
       <Platform>arm64</Platform>
     </OfficialBuildRID>
   </ItemGroup>
+  <ItemGroup Condition="$(SupportedPackageOSGroups.Contains(';Android;'))">
+    <OfficialBuildRID Include="android.21-arm64">
```

Comment:
@chcosta At this line? I don't think so…a quick check of master shows that the other supported OSs don;t include it at this point. Should it be?

@gkhanna79 Anything you see that needs changing?